geometrycollection.go

```diff
@@ -109,12 +113,52 @@ func (g *GeometryCollection) MustPush(gs ...T) *GeometryCollection {
 	return g
 }
 
+// MustSetLayout sets g's layout. It panics on any error.
+func (g *GeometryCollection) MustSetLayout(layout Layout) *GeometryCollection {
```

Comment:
minor nit: have this function call `SetLayout` and panic if it errors in the same way that `MustPush` calls `Push`